bin/spectacle/init.js

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const prompts = require('prompts');
+
+const mdRegex = /(\.mdx$|\.md$)/gm;
+const jsRegex = /(\.json$|\.js$)/gm;
+const jsxRegex = /(\.jsx$|\.js$)/gm;
+const { pathExists } = require('fs-extra');
+
+const questions = [
+  {
+    type: 'select',
+    name: 'action',
+    message: 'Create a live server or build directory?',
+    choices: [
+      { title: 'Server', value: 'server' },
+      { title: 'Build', value: 'build' }
+    ]
+  },
+  {
+    type: prev => (prev === 'server' ? 'number' : null),
+    name: 'port',
+    message: 'What port for the live server?',
+    validate: val => {
+      const numLength = val.toString().length;
```

Comment:
Ports can be lower than 1000 and higher than 9999 (although there's a max limit and low numbered ports require specific privileges). Maybe we just validate the number is greater than 0?